tests/testthat/test-formatters.R

```diff
@@ -537,3 +537,7 @@ mpf <- MatrixPrintForm(strings = strs, spans = spans, aligns = aligns,
 
 expect_equal(length(grep("spn_val", toString(mpf))),
              1L)
+
+test_that("default_hsep works as expected", {
+  expect_identical(default_hsep(), "—")
```

Comment:
This will fail on old systems which do not fully support UTF-8 (at least one of CRAN's buidl machines was such a system in the last cycle).

Also, this is a false positive. we have tests for `export_as_txt`, which calls default_hsep by default, so this actually is covered...

Reply:
fyi @pawelru @shajoezhu more false positives from autovalidate.

Reply:
thanks @gmbecker , hi @dgkf-roche fyi. Was wondering will autovalidater be updated?